src/components/common/view/Icon/Icon.js

```diff
@@ -15,6 +17,9 @@ type IconProps = {
   theme: Object,
 }
 
-export default withTheme(({ theme, color, size, ...props }: IconProps) => (
-  <Icon size={size || 16} color={theme.colors[color] || color || theme.colors.primary} {...props} />
-))
+export default withTheme(({ theme, color, size = 16, onPress = noop, ...iconProps }: IconProps) => {
+  const onIconPress = useOnPress(onPress)
```

Comment:
adding a default onpress changed all icons to have the pointer cursor (see snapshots), even if they are not clickable

Reply:
it also could cause buttons non-clickable outside an icon. it's a known bug, it was already fixed in master/react-native. i'll re-merge with master and will make sure the fix is applied

thanks for finding that